assets/js/components/Dashboard/TimePicker.tsx

```diff
@@ -13,28 +21,31 @@ interface TimePickerProps {
 const TimePicker = ({ selectedTime, onChange, id }: TimePickerProps) => {
   const [showOverlay, setShowOverlay] = useState(false);
   const ref = useRef(null);
-
   const selectedMoment = moment(selectedTime, "h:mm A");
 
   return (
     <>
-      <Form.Control
-        id={id}
-        ref={ref}
-        type="time"
-        value={selectedTime}
-        onChange={(input) =>
-          onChange(moment(input.target.value, "HH:mm").format("HH:mm"))
-        }
-        onKeyUp={(e) => {
-          if (e.key === "Enter") {
-            setShowOverlay(true);
+      <InputGroup className="time-picker" ref={ref}>
+        <Form.Control
+          id={id}
+          type="time"
+          value={selectedTime}
+          onChange={(input) =>
+            onChange(moment(input.target.value, "HH:mm").format("HH:mm"))
```

Comment:
What does this do? It appears we're parsing a time from the input value, then formatting it, but not doing anything with it after that.

Reply:
This is unnecessary. Can just pass the value directly into `onChange`.

Reply:
Oh, whoops, I was missing that this was wrapped in `onChange` which is a reference to the component's prop.